### PR TITLE
RR-354 update content for CIAG API being unavailable

### PR DESCRIPTION
**Pull request by chrimesdev**
## Description

Update content for CIAG API being unavailable on the Education and Training tab

https://dsdmoj.atlassian.net/browse/RR-354

Screenshot 2023-10-05 at 19 03 03

### CODE REVIEW

**Comment by chrimesdev** [comment]
Merge https://github.com/ministryofjustice/hmpps-education-and-work-plan-ui/pull/227 first - once this is merge the tests will pass.

File: server/views/pages/overview/partials/educationAndTrainingFunctionalSkills.njk [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
+      <div class="govuk-summary-card__title-wrapper">
+        <h2 class="govuk-summary-card__title">Current functional skills</h2>
+        <ul class="govuk-summary-card__actions govuk-!-display-none-print">
+            <li class="govuk-summary-card__action">
+              <a class="govuk-link" href="../functional-skills" data-qa="view-all-functional-skills-button">View all<span class="govuk-visually-hidden"> current functional skills</span></a>
+            </li>
+        </ul>
+      </div>
+      <div class="govuk-summary-card__content">
+        {% if not functionalSkills.problemRetrievingData %}
```
Comment (chrimesdev) [inline server/views/pages/overview/partials/educationAndTrainingFunctionalSkills.njk:14]: The `if statement` has moved to inside the summary card

**Comment by nathanrussell-moj-digital** [comment]
Minor question; now that the message about Curious being down is in the table (as per the requirement), I wonder if we should hide the "View all" button? I wonder if its worth a quick question to Aliki or Karen? Or whether we ignore it?
I'll approve the PR as is, and let you decide whats best to do 😁

**Review by nathanrussell-moj-digital — APPROVED** [review]

**Comment by chrimesdev** [comment]
> Minor question; now that the message about Curious being down is in the table (as per the requirement), I wonder if we should hide the "View all" button? I wonder if its worth a quick question to Aliki or Karen? Or whether we ignore it? I'll approve the PR as is, and let you decide whats best to do 😁

Good spot, I wonder if its worth another ticket and keep this one simple with content? I'll raise it with Aliki and Karen

**Pull request merged**